src/cozmo/objects.py

```diff
@@ -186,6 +197,28 @@ def _observed_timeout(self):
         self._is_visible = False
         self.dispatch_event(EvtObjectDisappeared, obj=self)
 
+    def _handle_available_object(self, available_object):
+        # triggered when engine sends available objects
+        # as a response to a RequestAvailableObjects message
+        if self.last_observed_robot_timestamp and \
+                (self.last_observed_robot_timestamp > available_object.lastObservedTimestamp):
```

Comment:
Better style to wrap the whole if condition in parentheses and drop the backslash at the end of the first line